### PR TITLE
Note on Time Sync for Agent and Backend

### LINKED ISSUE #1187: Document time synchronization requirement for Sensu Go
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)

## Expected Behavior

Docs should explicitly describe the need for time synchronization (using of NTP or similar) to ensure consistency of system clock time between backend(s) and agent(s).

## Current Behavior

Time synchronization requirements are described in the Sensu Core FAQ but do not seem to be covered in Sensu Go documentation.

## Context

From the [Sensu Core FAQ](https://docs.sensu.io/sensu-core/latest/faq/):

> When system clocks are out of sync between Sensu clients (where data is collected) and the Sensu server (where data is processed), Sensu may generate false positive client keepalive events, among other potentially unexpected behaviors.

### PR DESCRIPTION
**Pull request by Blue0ctober**
## Description

Closes #1187

## Motivation and Context

Adds notes re: Syncing agent and backend to NTP servers. 

## Review Instructions

- [ ] Does the note make sense?
- [ ] Does the note location make sense?

### CODE REVIEW

**Comment by apaskulin** [comment]
This PR requires a follow up to apply changes to version 5.3.

**Review by apaskulin — COMMENTED** [review]
Note-style callouts are best used to help users encountering edge cases or to clarify something confusing. Since the time sync requirement is an important part of the content here, I'd recommend including it as normal text under a  `### Time synchronization` subsection as the last subsection within the Operation section on both these pages.

File: content/sensu-go/5.0/reference/agent.md [hunk 1]
```diff
@@ -509,6 +510,10 @@ You can specify a deregistration handler per agent using the [`deregistration-ha
 
 Agents can connect to a Sensu cluster by specifying any Sensu backend URL in the cluster in the [`backend-url` configuration flag][16]. For more information about clustering, see [Sensu backend datastore configuration flags][35] and the [guide to running a Sensu cluster][36].
 
+### Time Synchronization
+
+Time clocks between agents and the backend should be synchronized to a central NTP server. Out of sync time clocks may cause issues with metric and check alerts.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/reference/agent.md]:
```suggestion
System clocks between agents and the backend should be synchronized to a central NTP server. Out of sync system time may cause issues with keepalive, metric and check alerts.
```

File: content/sensu-go/5.0/reference/backend.md [hunk 2]
```diff
@@ -151,6 +152,10 @@ To configure a cluster, see:
 - [Datastore configuration flags][12]
 - [Guide to running a Sensu cluster][13]
 
+### Time Synchronization
+
+Time clocks between agents and the backend should be synchronized to a central NTP server. Out of sync time clocks may cause issues with metric and check alerts.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/reference/backend.md]:
```suggestion
System clocks between agents and the backend should be synchronized to a central NTP server. Out of sync system time may cause issues with keepalive, metric and check alerts.
```

**Comment by cwjohnston** [comment]
In the backend content, let's add a link to etcd documentation on ntpd.

**Review by cwjohnston — DISMISSED** [review]

**Review by apaskulin — APPROVED** [review]

**Pull request merged**